virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -64,7 +64,9 @@ This release adds new features and enhancements related to the following compone
 
 //CNV-34757: Guest OS console logs are collected by default
 
-//CNV-34680: Metrics update. This is probably related to the note in the Deprecated features section.
+//CNV-34680: Metrics update. 
+* You can use the `kubevirt_vm_created_total` metric (Type: Counter) to query the number of VMs that are created in the specified namespace.
```

Comment:
`that are created` Perhaps `that were created`

Reply:
Will it be helpful to link to the [Querying metrics](https://docs.openshift.com/container-platform/4.13/virt/support/monitoring/virt-prometheus-queries.html#about-querying-metrics_virt-prometheus-queries) page in our documentation?

I was thinking we could rewrite the note as:

```suggestion
* You can use Prometheus to monitor the `kubevirt_vm_created_total` metric (Type: Counter). `kubevirt_vm_created_total` returns the total number of VMs created in the specified namespace.
```

Reply:
Updated the note text

Reply:
@sradco Updated the text per our conversation on Slack. Please review. Thanks!